modules/administration-guide/partials/proc_configuring-a-che-workspace-with-a-persistent-volume-strategy.adoc

```diff
@@ -100,12 +100,7 @@ When deploying {prod-short} using the Operator, configure the intended strategy
 
 .Procedure
 
-The following procedure steps are available for:
-
-* OpenShift command-line tool, `oc`
-ifeval::["{project-context}" == "che"]
-* Kubernetes clusters controlling command-line tool, `kubectl` 
-endif::[]
+The following procedure steps are available for {platforms-name} command-line tool, {platforms-cli}.
```

Comment:
Don't think it renders correctly: `The following procedure steps are available for Kubernetes or OpenShift  command-line tool, kubectl.`
kubectl has nothing to do with OpenShift

I also think it needs a dash or a colon instead of comma. WDYT @rkratky ?

Reply:
Shoule be rendered as: 

The following procedure steps are available for Kubernetes or OpenShift command-line tool, `kubectl` or `oc`.